web/templates/classEmma.class.php

```diff
@@ -541,6 +541,44 @@ function getClubResults($compId, $club)
 			return $ret;
 	}
 
+	function getPending($compId, $limit)
+	{
+		$ret = Array();
+		$q = "SELECT runners.Name, runners.Club, results.Time, runners.Class ,results.Status, results.DbID, results.Control ";
+		$q .= "FROM runners,results ";
+		$q .= "WHERE results.DbID = runners.DbId AND results.TavId = ". $this->m_CompId ." AND runners.TavId = ".$this->m_CompId ." AND results.Status in (9,10) ";

```

Comment:
Here I assume that all row in the results table have the actual `Status` set to 9 or 10 if the competitor is still pending. If the `Status` is only set for control 1000, the query would have to be adjusted accordingly.

Reply:
Yes, Status 9 and 10 is still pending and status is only set for finish (1000)